src/components/map/map.tsx

```diff
@@ -171,9 +195,27 @@ export class Map implements GxComponent {
     }
   }
 
+  private setUserLocation({ coords }) {
+    this.userLocationCoords = `${coords.latitude}, ${coords.longitude}`;
+    this.userLocationChange.emit(this.userLocationCoords);
```

Comment:
If the `userLocationChange` will be fired every time the `userLocationCoords` property is changed, we can use the `@Watch` decorator to watch changes on the `userLocationCoords` property and fire the event there.

That way, we will be sure that every time the property is updated, the event will be fired.

Reply:
Perhaps I didn't understand... Look:
Firing ```setUserLocation``` method **do not** depends of changing ```userLocationCoords``` property.  ```setUserLocation``` method is fire by the Geolocation ```watchPosition``` method.
Geolocation ```watchPosition``` method is fired, calls ```setUserLocation``` method to refresh the coords of "_userLocationMarker_" and emit ```userLocationChange``` with the coords of "_userLocationMarker_"

Reply:
What I propose is firing the `userLocationChange` event every time the `userLocationCoords` property is changed. This can be achieved using the `@Watch` decorator.

There's nothing wrong with your current approach, I think it's quite good. However, thinking about the maintainability of the code, using the `@Watch` decorator makes a lot of sense, because it will ensure that, if someone in the future modifies the code and changes the `userLocationCoords` property in another place,, the event will be fired.

Reply:
Check changes in bf6f502 and tell me what do u think 😉